### PR TITLE
Update initializers CHIP

### PR DESCRIPTION
**Pull request by lydia-duncan**
- Fix a reference to a "constructor" when I meant "initializer"
- Fix the outline so that it doesn't group the sub-bullets of "Defining the
  Initializer Body" weirdly
- Specify that if statements used around calls to parent or sibling initializers
  must be param
- Add a section explicitly stating where an initializer may be defined and ways
  to prevent outsiders from defining unapproved initializers on a type if the
  type designer is worried about such things (as discussed in a previous
  meeting)

### CODE REVIEW

File: doc/chips/10.rst [hunk 1]
```diff
@@ -163,13 +165,13 @@ earlier fields, but not of later fields:
 fields may be accessed during this phase, as they have not been given an initial
 value yet – the parent's Phase 1 will be entered once the child's phase 1 is
 complete (more information on this process will be provided later in this
```
Comment (vasslitvinov) [inline doc/chips/10.rst]: There are fewer actual changes here than meets the eye. :)

File: doc/chips/10.rst [hunk 2]
```diff
@@ -223,14 +225,14 @@ The first syntax has the benefit of maintaining the initializer as a single
...
-both phases.  However, since the call to the parent constructor serves as the
-division between the two phases, it would be easy for this statement to get lost
-amid a larger and more complex initializer body.  Additionally, because the
-phase split is not as extreme as in the second syntax, the type designer may be
-more confused or frustrated when code placed after the call is valid but
-identical code before it is not.
+shared from Phase 1 to Phase 2, and ``param`` or compile-time const ``if``
+statements may be used to wrap across both phases, though loops and parallel
+statements are not allowed to encapsulate both phases.  However, since the call
+to the parent constructor serves as the division between the two phases, it
```
Comment (vasslitvinov) [inline doc/chips/10.rst]: how about constructor->initializer

Comment (lydia-duncan) [inline doc/chips/10.rst]: Shoot!  I missed that one in my review.  Thanks for catching!

**Comment by vasslitvinov** [comment]
Other than the above comment, looks good to go!

**Pull request merged**